### PR TITLE
Update LayerStackup.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/modules/LayerStackup.py [hunk 1]
```diff
@@ -78,11 +79,26 @@ def getIfromRGB(rgb):
...
 class Layer(object):
-    """Layer class."""
+    """Layer class.
+    
+    Parameters
+    ----------
+    editor :
+    
+    type : string, optional
+        The defualt is ``"signal"``.
```
Comment (PipKat) [inline pyaedt/modules/LayerStackup.py:89]:
```suggestion
        The default is ``"signal"``.
```

File: pyaedt/modules/LayerStackup.py [hunk 2]
```diff
@@ -12,12 +12,12 @@
...
 @aedt_exception_handler
 def str2bool(str0):
-    """
+    """Convert a string to a Boolean value.
 
     Parameters
     ----------
-    str0 :
-        
+    str0 : text
```
Comment (akaszynski) [inline pyaedt/modules/LayerStackup.py:19]:
```suggestion
    str0 : str
```
It's a bad parameter name and that probably threw you off.

File: pyaedt/modules/LayerStackup.py [hunk 3]
```diff
@@ -32,11 +32,12 @@ def str2bool(str0):
         return ""
 
 def conv_number(number, typen=float):
-    """
+    """Convert a number.
 
     Parameters
     ----------
     number : float
```
Comment (akaszynski) [inline pyaedt/modules/LayerStackup.py:39]:
```suggestion
    float
```

File: pyaedt/modules/LayerStackup.py [hunk 4]
```diff
@@ -32,11 +32,12 @@ def str2bool(str0):
...
+       Number to convert.
```
Comment (akaszynski) [inline pyaedt/modules/LayerStackup.py:40]:
```suggestion
       Number represented a float.
```

File: pyaedt/modules/LayerStackup.py [hunk 5]
```diff
@@ -160,7 +176,7 @@ def set_layer_color(self, r, g, b):
         r : int
             Red color value.
         g : int
-            Green color value.
+            Green color vale.
```
Comment (akaszynski) [inline pyaedt/modules/LayerStackup.py:179]:
```suggestion
            Green color value.
```

**Review by akaszynski — APPROVED** [review]
Approved with suggestions.

**Pull request merged**